os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/trace_uart.c

```diff
@@ -90,46 +90,40 @@ uint32_t traceuart_irq(void *data)
 
 bool trace_uart_init(void)
 {
-    if(!CHECK_SW(EFUSE_SW_TRACE_SWITCH))
+    if (!CHECK_SW(EFUSE_SW_TRACE_SWITCH))
     {
-        /* 0 */
-    platform_debug("trace_uart_init:  TRACE OPEN");
-    g_uart_obj.tx_switch = true;
+        printf("trace_uart_init: TRACE OPEN\r\n");
```

Comment:
We don't recommend `printf` in non-app because `printf` can't be configured by module or level.
I think we need to use `Xdbg` defined in `<debug.h>` instead of printf in non-app.
If you use `Xdbg`, you can configure by level, error, warning, or info and enable or disable log printing by menuconfig. 
As I checked, there is no debug functions for bluetooth.
Could you please define debug functions for bluetooth and update all usages of printf with another commit?
You can refer this page for `debug messages` and other functions defined in `<debug.h>`
https://github.com/Samsung/TizenRT/blob/master/docs/HowToUseLoggingSystem.md#debug-messages

Reply:
It is okay to commit all the changes for debug log in this PR? or should raise another PR for debug log changes?
We used debug_print which is define as printf, it is control by RTK_DEBUG_ON define.
Maybe I should raise another PR to fix all those. This PR to just modify the changes of printf involve.

Reply:
I have change current modified file for this PR.

Reply:
Thank you for update! I didn't mean to fix every printf here.
Please update what I said with another PR. Thank you :)